acq4/pyqtgraph/widgets/SpinBox.py

```diff
@@ -143,7 +143,7 @@ def setOpts(self, **opts):
         """
         #print opts
         for k in opts:
-            if k == 'bounds':
+            if k == 'bounds' or k == 'range':
```

Comment:
I think you're right, but there are a lot of places (like in all the flowchart nodes) where we expect 'range' to be a valid option. These all probably failed silently in the past. Adding 'range' as an option to SpinBox seemed like a reasonable thing. Alternatively, we could change all the places we pass in range to use bounds instead. Thoughts?